components/nodes/node-stanford-news.tsx

```diff
@@ -1,46 +1,104 @@
+import { FacebookIcon } from '@/components/simple/icons/FacebookIcon';
+import { TwitterIcon } from '@/components/simple/icons/TwitterIcon';
+import { LinkedInIcon } from '@/components/simple/icons/LinkedInIcon';
+import {Card} from "@/components/patterns/card";
+import { NewsArticleJsonLd } from 'next-seo';
 
 interface NewsNodeProps {
   node: News
 }
 
 export const NodeStanfordNews = ({node, ...props}: NewsNodeProps) => {
 
+  const [currentUrl, setCurrentUrl] = useState("#")
+  useEffect(() => {
+    setCurrentUrl(document.URL);
+  }, [])
+
   return (
     <MainContentLayout>
-      <NextSeo
-        title={node.title}
-        canonical={node.su_news_source?.url}
-      />
-
-
-      <article {...props}>
-        {node.su_news_topics && node.su_news_topics.map(topic =>
-          <div key={topic.id}>
-            {topic.name}
-          </div>
+      <article {...props} className="su-mt-50">
+        <NewsArticleJsonLd
+            url={node.su_news_source?.url}
+            title={node.title}
+            images={node.su_news_featured_media?.field_media_image?.uri?.url}
+            section=""
+            keywords=""
+            dateCreated={node.su_news_publishing_date}
+            datePublished={node.su_news_publishing_date}
+            dateModified=""
+            authorName={node.su_news_byline}
+            description={node.su_news_dek}
+            body=""
+            publisherName=""
+            publisherLogo=""
+            isAccessibleForFree={true}
+        />
+        {node.su_news_topics && node.su_news_topics.map((topic, index) =>
+            <span key={topic.id} className="su-text-digital-red su-font-semibold">
+              {(index ? ', ' : '') + topic.name} 
+            </span>
         )}
-
-        <h1>{node.title}</h1>
-        {node.su_news_dek && <div>{node.su_news_dek}</div>}
-
-        {node.su_news_publishing_date && <div>{formatDate(node.su_news_publishing_date + ' 12:00:00')}</div>}
-        {node.su_news_byline && <div>{node.su_news_byline}</div>}
+        <h1 className="su-type-5">{node.title}</h1>
+        {node.su_news_dek && <div className="su-rs-mb-1">{node.su_news_dek}</div>}
+        <div className="md:su-flex su-rs-mb-7">
+          <div className="su-flex md:su-order-last su-rs-mb-2">
+            <ul className="su-flex su-list-unstyled md:su-pl-[10px] su-mt-[-3px]">
+              <li className="su-mr-1em">
+                <Link className="su-text-black hocus:su-text-facebook su-transition-colors" href={`http://www.facebook.com/sharer.php?u=${currentUrl}&display=popup`}>
+                  <span className="su-sr-only">Stanford Facebook</span>
+                  <FacebookIcon />
+                </Link>
+              </li>
+              <li className="su-mr-1em">
+                <Link className="su-text-black hocus:su-text-twitter su-transition-colors" href={`https://twitter.com/intent/tweet?url={window.location}&text=${currentUrl}`}>
```

Comment:
Shouldn't url be the `currentUrl` and the 5ext be the title?

Reply:
ah good catch, thanks